src/ggrc-client/js/components/object-selection/object-selection-item.js

```diff
@@ -15,6 +15,7 @@ export default can.Component.extend({
     item: null,
     isDisabled: false,
     isSelected: false,
+    isMega: false,
```

Comment:
```suggestion
    isBlocked: false,
```
Let's introduce a new state for `object-seletion-item`: `isBlocked` which will mean that item is disabled and unchecked.

Reply:
done